nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestColdStorage.java

```diff
@@ -90,23 +97,34 @@
 
     @Test
     public void shouldMoveToColdStorage() throws IOException {
-        DocumentModel documentModel = createFileDocument(DEFAULT_DOC_NAME, true);
+        // with regular user with "WriteColdStorage" permission
+        ACE[] aces = { new ACE("john", SecurityConstants.READ, true), //
+                new ACE("john", SecurityConstants.WRITE, true), //
+                new ACE("john", SecurityConstants.WRITE_COLD_STORAGE, true) };
+        DocumentModel documentModel = createFileDocument(DEFAULT_DOC_NAME, true, aces);
+
+        try (CloseableCoreSession userSession = CoreInstance.openCoreSession(documentModel.getRepositoryName(),
+                "john")) {
+            moveAndVerifyContent(userSession, documentModel);
+        }
 
+        documentModel = createFileDocument(DEFAULT_DOC_NAME, true, aces);
         // move the blob to cold storage
-        documentModel = ColdStorageHelper.moveContentToColdStorage(session, documentModel.getRef());
-        session.saveDocument(documentModel);
-        transactionalFeature.nextTransaction();
-        documentModel.refresh();
-
-        assertTrue(documentModel.hasFacet(FacetNames.COLD_STORAGE));
-
-        assertNull(documentModel.getPropertyValue(ColdStorageHelper.FILE_CONTENT_PROPERTY));
+        moveAndVerifyContent(session, documentModel);
```

Comment:
If I'm not mistaken, this second case is for the Administrator user, so:
- it doesn't need aces when creating the document
- it deserves a comment above like "// with administrator user"